### PR TITLE
hwdef: support rFCU H743 flight controller

### PR DESCRIPTION
**Pull request by tridge**

### CODE REVIEW

**Comment by MallikarjunSE** [comment]
Are multiple heaters handled individually? What would be the temperature reference for each heater?

**Comment by tridge** [comment]
> Are multiple heaters handled individually? What would be the temperature reference for each heater?

on this board both GPIOs must be activated for the heater to work

**Review by bugobliterator — APPROVED** [review]

File: libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 #define SENSOR_RATE_DEBUG 0
 
+#ifndef AP_IMU_HEATER_INSTANCE
```
Comment (tridge) [inline libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp:15]: change to AP_HEATER_IMU_INSTANCE

**Review by peterbarker — APPROVED** [review]
LGTM past the define rename

File: libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp (hunk 1 above)
Comment (tridge) [reply libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp]: changed

**Comment by Hwurzburg** [comment]
@tridge we continue to merge new boards without a README.md or sufficient README....this only indefinitely delays its addition to the wiki

**Comment by Hwurzburg** [comment]
@MallikarjunSE without a proper README.md file adding this to the wiki is indefinitely delayed...fyi

**Comment by MallikarjunSE** [comment]
> @MallikarjunSE assuming this is Serra's bd (big if) just letting you know without a proper README.md file adding this to the wiki is indefinitely delayed...fyi

Hey @Hwurzburg 
This board is not related to me. I'll add wiki to old Sierra boards soon. 
Thank you!

**Pull request merged**